Add release notes for 1.11.4 (#51)
anbox-cloud/discourse-docs: announcements/1.11.4.md

### Introduction

The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.11.4.

Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.

#### New features & improvements

> None

Reviewer: ```suggestion
None.
```

#### Bugs

> A bug in Anbox causes the random webview based application crashes when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgrade the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability and are planning to provide the fix in the Anbox Cloud 1.12 release.

Reviewer: ```suggestion
A bug in Anbox caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.
```

Reviewer: Just to confirm: A bug in **Anbox** or a bug in **Anbox Cloud**?

Author: hmm, I think it's better to call out *Anbox* here than *Anbox Cloud* as it's caused by the lower stack. Maybe @morphis has a better idea.

Reviewer 2: Lets call it the `Anbox runtime` here. Anbox Cloud is the entire stack whereas Anbox is one particular thing in the stack. Calling it `Anbox runtime` should make the distinction clear.

Author: +1

> A bug in Anbox runtime caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.

Reviewer: ```suggestion
A bug in the Anbox runtime caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.
```
(In both files)

#### Upgrade instructions

> This release does not include any charm changes. Only images are updated and will be automatically synchronized to your Anbox Cloud deployment via the Canonical image server. See (Image Management)[https://anbox-cloud.io/docs/howto/manage/images]  for more details.

Reviewer: ```suggestion
This release does not include any charm changes. Only the images are updated, and they are automatically synchronised to your Anbox Cloud deployment via the Canonical image server. See [Manage images](https://anbox-cloud.io/docs/howto/manage/images) for details.
```
